share/container-build.sh

```diff
@@ -2,17 +2,13 @@
 
 #
 # SPDX-FileCopyrightText:  2023, Iker Pedrosa <ipedrosa@redhat.com>
+# SPDX-FileCopyrightText:  2024, Iker Pedrosa <ipedrosa@redhat.com>
 #
 # SPDX-License-Identifier:  BSD-3-Clause
 #
 
-for FILE in share/containers/*; do
-	IFS='/'
-	read -ra ADDR <<< "$FILE"
-	IFS='.'
-	read -ra ADDR <<< "${ADDR[2]}"
-	IFS=''
-	if ! docker build -f $FILE . --output build-out/${ADDR[0]}; then
-		exit
-	fi
-done
+pushd ansible/
```

Comment:
pushd(1) without popd(1) is just cd(1)

Reply:
Not really. This is a script and everything that runs inside it it's affected by `pushd`, but as soon as the script finishes the state is recovered. If I want to do the same with `cd` I have to run it again at the end to recover the state.

Reply:
Please show the difference with an example; I'm not seeing it.

```sh
$ pwd
/home/alx/tmp/shadow
```
```sh
$ ls
cd.bash  pd.bash
```
```sh
$ cat cd.bash 
	cd /tmp
	pwd
	ls
```
```sh
$ cat pd.bash 
	pushd /tmp
	pwd
	ls
```
```sh
$ bash cd.bash 
/tmp
Temp-988ac446-4a36-4ca3-a295-a819768048fd
grepc.patterns.Di6KCW
grepc.patterns.nvO5bb
ssh-mEQv6ojqkOKI
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-ModemManager.service-NfEV9r
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-colord.service-hnzWJu
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-exim4.service-vGmQKr
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-polkit.service-TdhxGF
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-switcheroo-control.service-N089gp
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-systemd-logind.service-ILr6Ix
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-systemd-timesyncd.service-GvIeWo
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-upower.service-wSe2ke
```
```sh
$ pwd
/home/alx/tmp/shadow
```
```sh
$ bash pd.bash 
/tmp ~/tmp/shadow
/tmp
Temp-988ac446-4a36-4ca3-a295-a819768048fd
grepc.patterns.Di6KCW
grepc.patterns.nvO5bb
ssh-mEQv6ojqkOKI
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-ModemManager.service-NfEV9r
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-colord.service-hnzWJu
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-exim4.service-vGmQKr
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-polkit.service-TdhxGF
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-switcheroo-control.service-N089gp
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-systemd-logind.service-ILr6Ix
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-systemd-timesyncd.service-GvIeWo
systemd-private-2c58461cd7dd4e95939ee9a5b8df6f39-upower.service-wSe2ke
```
```sh
$ pwd
/home/alx/tmp/shadow
```

Reply:
You are right, I changed it to `cd`